testing/test_config.py

```diff
@@ -40,6 +43,50 @@ def test_simple_producer():
     app.start()
 
 
+def test_handover_conf():
+
+    val_source = {
+        "nodes": {
+            "node1": {
```

Comment:
I'm pretty sure the  format here is defined by the imported configurator code and I'm not going to prioritise spending much time fixing it.